src/app/core/ui/search/search.component.html

```diff
@@ -8,6 +8,7 @@
     <mat-option *ngFor="let res of results" [value]="''">
       <app-child-block *ngIf="res.getType()==='Child'" [entity]="res" [ngClass]="{'result-inactive': !res.isActive()}"></app-child-block>
       <app-school-block *ngIf="res.getType()==='School'" [entity]="res"></app-school-block>
+      <app-search-information *ngIf="res.getType()==='SearchInformation'" [entity]="res"></app-search-information>
```

Comment:
I used the `app-child-block` and `app-school-block` components to build on the reusable presentation of these Entities. for displaying the search information text, that is only relevant within the SearchComponent you could even just directly display it here, maybe using a html `span`. A separate component can also be nice (not sure whether it's an overkill here).

It might be easier to keep that search-information separate from the results array and it's *ngFor, then you don't have to make sure it has the same format like entities with their getType(), etc.
e.g. `<mat-option *ngIf="searchInformationText"><span class="search-information">{{searchInformationText}}</span></mat-option>`